src/object/srv_obj.c

```diff
@@ -1053,6 +1053,46 @@ obj_fetch_create_maps(crt_rpc_t *rpc, struct bio_desc *biod)
 	return 0;
 }
 
+static int
+obj_fetch_shadow(daos_handle_t coh, daos_unit_oid_t oid, daos_epoch_t epoch,
+		uint64_t cond_flags, daos_key_t *dkey, unsigned int iod_nr,
+		daos_iod_t *iods, uint32_t tgt_idx,
+		struct daos_recx_ep_list **pshadows)
+{
+	struct daos_oclass_attr		*oca;
+	daos_handle_t			 ioh = DAOS_HDL_INVAL;
+	int				 rc;
+
+	obj_iod_idx_vos2parity(iod_nr, iods);
```

Comment:
style: call this after the if (oca == NULL...)?

Reply:
OK, I'll change it.